### PR TITLE
Bugfix mouseReleased

### PR DESCRIPTION
**Pull request by oseph**
This should address #181 where `mouseReleased()` misreports all events with a `mouseButton` value of `0`

As well, `ctrl-leftClick` for macOS users should be properly recognized as a secondary(right) click.

I also updated PConstants to include `MOUSE_LEFT, MOUSE_MIDDLE, MOUSE_RIGHT` instead of the `LEFT,CENTER,RIGHT` constants. The current constants are associated with keyboard events such as `KeyEvent.VK_LEFT` and returned values that didn't make much sense, e.g. left click returning a mouseButton value of 37?

In short, mouseButton returns 1,2,3 (leftclick ,middleclick, rightclick).

As far as I can tell on my end, everything behaves properly for sketches using the default renderer, P2D, and P3D. I've only tested this on macOS (11.2.3). Would love it if anyone out there using Linux/Windows could confirm things behave on their end.

### CODE REVIEW

**Comment by benfry** [comment]
Thanks for the attempt. As someone mentioned in #181, definitely better if you can submit patches for the issue at hand without extra changes. For instance, the short constant names (`LEFT`, `RIGHT`, `CENTER`) are a feature, not a bug—you'll notice a similar level of brevity throughout the entire API. And regardless, I can't just rename them—it would break code and examples for many, many thousands of people.
